roles/telemetry_autoscaling/tasks/set_env_var.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Set environment variables for JUNIT callback plugin.
+# It writes playbook output to a JUnit formatted XML file.
+- name: envt vars
+  set_fact:
+    os_environment:
+      - key: JUNIT_TEST_CASE_PREFIX
```

Comment:
we don't need to add it to testsuites , it will make it filter only tasks start with that